hpp-fcl.tf

```
@@ -0,0 +1,19 @@
+locals {
+  hpp_fcl_team = [
+    "jcarpent",
+    "jmirabel",
+    "wxmerkt",
+    "nim65s",
+  ]
+  hpp_fcl_repositories = [
+    "hpp-fcl-release",
+  ]
+}
+  
+module "hpp_fcl_team" {
+  source       = "./modules/release_team"
+  team_name    = "hpp-fcl"
```

```suggestion
  team_name    = "hpp_fcl"
```

I'll suggest that we use `_` in the team, repository, and terraform file names to match the ROS package naming conventions.